standard-covoiturage_openapi.yaml

```diff
@@ -0,0 +1,506 @@
+          headers:
+            Retry-After:
+              description: |-
+                How long to wait before making a new request (in seconds).
+              schema:
+                type: integer
+        500:
+          description: Internal Server Error. Please try again later.
+
+#
+# Schemas 
+#              
+components:
+
+  parameters:
+  
+    departureLat:
+      name: departureLat
+      in: query
+      required: true
+      schema:
+        type: number
+      description: Latitude of searched departure point.
+      
+      
+    departureLng:  
+      name: departureLng
+      in: query
+      description: Longitude of searched departure point.
+      required: true
+      schema:
+        type: number
+    
+    arrivalLat:
+      name: arrivalLat
+      in: query
+      description: Latitude of searched arrival point.
+      required: true
+      schema:
+        type: number
+    
+    arrivalLng:    
+      name: arrivalLng
+      in: query
+      description: Longitude of searched arrival point.
+      required: true
+      schema:
+        type: number
+        
+    departureDate:
+      name: departureDate
+      in: query
+      description: Departure datetime using a UNIX UTC timestamp in seconds. 
+      schema:
+        type: integer
+            
+    timeDelta:
+      name: timeDelta
+      in: query
+      description: Time margin in seconds. The retrieved journeys must match the
+        given time parameters within a +`timeDelta` / -`timeDelta` interval .
+      schema:
+        type: integer
+        default: 900
+        
+    departureRadius:
+      name: departureRadius
+      in: query
+      description: Search radius in kilometers around the departure point.
+      schema:
+        type: number
+        default: 1.0
+        
+    arrivalRadius:
+      name: arrivalRadius
+      in: query
+      description: Search radius in kilometers around the arrival point.
+      schema:
+        type: number
+        default: 1.0
+        
+    count:
+      name: count
+      in: query
+      description: Maximum number of returned journeys. If missing, all matching
+        journeys are returned.
+      schema:
+        type: integer
+        
+  schemas:
+    Journey:
+      title: Journey
+      required:
+      - id
+      - duration
+      - passengerPickupDate
+      - passengerPickupLat
+      - passengerPickupLng
+      - passengerDropLat
+      - passengerDropLng
+      - type
+      - webUrl
+      type: object
+      properties:
+        id:
+          type: string
+          minLength: 1
+          maxLength: 255
+          description: Journey's id.
+            In order to be a basis for carpool proof, this id must match the `journey_id` that could be provided to a carpool proof register as described for example in [this documentation of an open source product for France](https://tech.covoiturage.beta.gouv.fr/partners/preuves/schema.html#schema-json-complet).
```

Comment:
I agree this would be better to avoid any such hard reference if possible. What about adding recommendations about linking to an existing scheme, with the example of RPC in France?
Or you could add a variable that references the external scheme that is used, so that in France we would make use of scheme=RPC but in a different context a different scheme could be used?